src/server/JasmineGraphServer.cpp

```diff
@@ -2557,27 +2557,157 @@ int JasmineGraphServer::initiateEntityResolution(vector<pair<string, string>> ho
                     JasmineGraphServer::collectBucketsToMaster(coordinatorHost, coordinatorPort, designatedWorkerHost,
                                                                designatedWorkerPort, designatedWorkerPort, graphID, 4,
                                                                masterIP);
+                    loop = true;
                 }
 
             }
 
         }
+        //TODO: Wait for coordinator to give instruction to share cluster files
+        loop = false;
+        while(!loop) {
+            bzero(data, 301);
+            read(sockfd, data, 300);
+            response = (data);
+            response = utils.trim_copy(response, " \f\n\r\t\v");
+            if (response.compare(JasmineGraphInstanceProtocol::SHARE_CLUSTER_FILES) == 0) {
+                loop = true;
+                int clusterCount = 4;
```

Comment:
Shall we get the clusterCount from Frontend?

Reply:
Cluster count changed to be based on worker count as discussed

Reply:
Ok